src/views/util.ts

```diff
@@ -0,0 +1,16 @@
+import { deleteComponent, getInteractiveFrame } from "../lib/codapPhone";
+import { TransformationDescription } from "../transformerStore/types";
+
+export async function closePlugin(
+  activeTransformations: Record<string, TransformationDescription[]>
+): Promise<void> {
+  if (
+    Object.keys(activeTransformations).length === 0 ||
```

Comment:
This doesn't quite work to check if there are any active transformations, since the keys are just input names, and are not deleted when transformations get deleted. I suggest

```typescript
function hasActiveTransformation(activeTransformations: ActiveTransformations) {
  return Object.values(activeTransformations).some((a) => a.length > 0);
}
```

Everything else looks good!

Reply:
Good catch!